src/animate/Scene.js

```diff
@@ -0,0 +1,70 @@
+import load from './load';
+import sound from './sound';
+
+/**
+ * Extends the PIXI.Application to provide easy loading.
```

Comment:
the PIXI.Application _class_ - the grammar is a little weird there without the last word.